source/extensions/filters/network/well_known_names.h

```diff
@@ -59,6 +59,8 @@ class NetworkFilterNameValues {
   const std::string ZooKeeperProxy = "envoy.filters.network.zookeeper_proxy";
   // WebAssembly filter
   const std::string Wasm = "envoy.filters.network.wasm";
+  // Sip proxy filter
+  const std::string SipProxy = "envoy.filters.network.sip_proxy";
```

Comment:
We are trying to remove this file, so please don't add this and just inline the string where needed.

Reply:
Done. Move to other file.